package.json

```diff
@@ -41,8 +41,7 @@
     "trailingComma": "es5"
   },
   "devDependencies": {
-    "babel": "^6.5.2",
-    "babel-cli": "^6.22.2",
+    "babel-cli": "^6.26.0",
```

Comment:
Why was this bump needed?

Reply:
I just ran two commands that Babel told me to. This was the byproduct of that.

Reply:
I was curious as to whether something fails without doing this upgrade?
We have lots of deprecation warnings, etc that could possibly be wrapped up in a separate PR.

Reply:
I couldn't run tests locally without making this change.

Reply:
:+1: